extensions/amp-ad-network-fake-impl/0.1/data/fake_amp_ad_with_iframe_transport.html

```diff
@@ -41,6 +37,37 @@
         }
       </script>
     </amp-analytics>
+    <amp-ad-exit id='exit-api'>
+      <script type='application/json'>
+        {
+          "targets": {
+            "target1": {
+              "finalUrl": "/?product1&x=CLICK_X&y=CLICK_Y&e=_elem&shouldNotBeReplaced=AMP_VERSION",
+              "vars": {
```

Comment:
It's used as an example to customers for how to implement. I do intend for it to include the new variable substitution...but was planning to add that along with adding the mechanism for the creative to send its response. My intention is to do those two things in a separate PR. I just added a line item to the design doc so that this doesn't fall through the cracks.